pkg/kit/kit.go

```diff
@@ -0,0 +1,95 @@
+/*
+ * TencentBlueKing is pleased to support the open source community by making
+ * 蓝鲸智云 - 配置平台 (BlueKing - CMDB) available.
+ * Copyright (C) 2025 Tencent. All rights reserved.
+ * Licensed under the MIT License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://opensource.org/licenses/MIT
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ * We undertake not to change the open source license (MIT license) applicable
+ * to the current version of the project delivered to anyone in the future.
+ */
+
+// Package kit is designed to manage and store business metadata, context and trace info
+package kit
+
+import (
+	"context"
+	"runtime"
+	"strings"
+	"time"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/trace"
+
+	"github.com/TencentBlueKing/bk-cmdb/pkg/log"
+)
+
+// Metadata the biz metadata
+type Metadata struct {
+	User     string // 操作人
+	AppCode  string // 来源AppCode
+	TenantID string // 来源多租户ID
+}
+
+// Kit a biz metadata and context kit
+type Kit struct {
+	context.Context
+	Metadata
+}
+
+// WithCancel same as context.WithCancel but return Kit
+func WithCancel(kt *Kit) (*Kit, context.CancelFunc) {
+	ctx, cancel := context.WithCancel(kt)
```

Comment:
这里是不是直接用kt.Context 会更更好
```suggestion
	ctx, cancel := context.WithCancel(kt.Context)
```

Reply:
下同

Reply:
done